graylog2-server/src/main/java/org/graylog2/events/ClusterEventPeriodical.java

```diff
@@ -86,16 +86,18 @@ static DBCollection prepareCollection(final MongoConnection mongoConnection) {
 
         DBCollection coll = db.getCollection(COLLECTION_NAME);
 
+        /* FIXME: Not supported by Fongo
```

Comment:
What's the plan with this? Can we just drop the warning? I don't know why it's a problem when the collection is capped.. :wink: